mvvmfx/src/main/java/de/saxsys/mvvmfx/FxmlPath.java

```diff
@@ -0,0 +1,38 @@
+package de.saxsys.mvvmfx;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation is used to define a {@link FxmlView} custom
+ * FXML file path. An empty file paths will be ignored.
+ *
+ * Note that the full path to the FXML file must be provided.
+ *
+ * Example: <br>
+ *     <br>
+ * <pre>
+ * package example.view;
+ *
+ *{@literal @}FxmlPath("/fxml/CustomPathView.fxml")
+ * public class CustomView implements {@link FxmlView} {
+ *
+ *         ...
+ *
+ * }
+ * </pre>
+ *
```

Comment:
I'd like to add a paragraph that makes it clear that this annotation only effects the parent view and not sub-views that are included via `<fx:include src=.../>` tag. I propose the following text:

```
Please be aware that this annotation only effects the parent view that is loaded by the {@link FluentViewLoader}.
Views that are included via "<fx:include ... />" tag aren't affected because the path is then detemined by the value of the "src" attribute in the include-tag. 
```